pkgs/by-name/is/iscc/package.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, fetchurl
+, innoextract
+, runtimeShell
+, wineWow64Packages
```

Comment:
Does it break with default version (wineWowPackages)? (If so, maybe add a comment?)

Reply:
Oh, I thought the 64 was a version number (but it's a "mode"). Sorry for the noise.